modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/TaskHeaderActions.tsx

```diff
@@ -83,11 +97,19 @@ const TaskHeaderActions = () => {
 
 				<ClayButton
 					displayType="secondary"
-					onClick={() =>
-						testrayTaskImpl.abandon(testrayTask).then(mutateTask)
-					}
+					onClick={() => {
+						const fn = subTaskAllCompleted
```

Comment:
¨fn" doesn't sound good to me.
I believe you can find a better name for this const.

Reply:
Hi @caiosouza15 this case it's fine, because `fn` is commonly used when we create a function to wrap a conditional.

But nice catch with the name 🗡️